### PR TITLE
[Blog] Added Remote Objects blog

### PR DESCRIPTION
**Pull request by aFinchy**
Signed-off-by: Amy Finch <amy@finchstudio.ca>

## Change summary

Created the blog for Remote Objects. Looking for an editorial as well as technical review. 

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Comment by aFinchy** [comment]
Screenshot 2022-11-23 at 14-59-47 O3DE Remote object support in Open 3D Engine

File: content/blog/posts/remoteobjects.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    "restricted": "RemoteGame"
+}
+```
+
+The important field here is the origin_uri field which is the location of the object. Currently for GitHub repositories that we want to sync this will usually be the same as the remote source url.
+
+The repo.json is placed in the same location as project.json in the root of the git repository.
+
+
+### Downloading using Project Manager
```
Comment (FiniteStateGit) [inline content/blog/posts/remoteobjects.md:73]:
```suggestion
### Download with Project Manager
```

File: content/blog/posts/remoteobjects.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+{{< image-width src="/images/blog/remote-objects/dropdown.PNG" width="100%" alt="Add a Remote Project" >}}
+
+Selecting this brings up the Add a remote project dialog. On entering a valid O3DE repositoriy URL the dialog updates and selects a download folder based on the name of the project and the default projects folder.
+
+{{< image-width src="/images/blog/remote-objects/addremotedialog.PNG" width="100%" alt="Add a Remote Project Config" >}}
+
+There is also the option whether to add the project to the build queue once it has downloaded. On selecting Download the project will be downloaded, registered and available to build and run like any other project.
+
+### Downloading using the O3DE CLI
```
Comment (FiniteStateGit) [inline content/blog/posts/remoteobjects.md:85]:
```suggestion
### Download with the O3DE CLI
```

File: content/blog/posts/remoteobjects.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+It is also possible to carry out these steps using the O3DE CLI. First by registering the remote source
```
Comment (FiniteStateGit) [inline content/blog/posts/remoteobjects.md:87]:
```suggestion
You can also add a remote project with the O3DE CLI. First, register the remote source:
```

File: content/blog/posts/remoteobjects.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+```powershell
+scripts\o3de.bat register -ru <remote source URL>
+```
+
+Then by specifying the project to download by the project name.
```
Comment (FiniteStateGit) [inline content/blog/posts/remoteobjects.md:93]:
```suggestion
Then, specify the project to download by the project name:
```

**Review by AMZN-Phil — APPROVED** [review]

**Review by FiniteStateGit — APPROVED** [review]

**Pull request merged**